### DISCUSSION TITLE
FlareSolverr to fix CloudFlare issues with Prowlarr indexers

### DISCUSSION THREAD

**Discussion by AdmiralAwesome** [tteck/Proxmox#2571] [Ideas]
Hey,

it would be a good idea to implement https://github.com/FlareSolverr/FlareSolverr - that one will help fixing problems with Indexers that Prowlarr/Radarr/Sonarr are using (for example 1337x).
![obraz](https://github.com/tteck/Proxmox/assets/2470577/e07c3b66-0813-4b66-9a45-f6c33a8dfc3d)

**Comment by navels** [comment]
FYI I added FlareSolverr to my Jackett container, was pretty easy: https://github.com/tteck/Proxmox/discussions/2495#discussioncomment-8504030

**Reply by AdmiralAwesome** [reply]
Oh, it's super easy to deploy on Docker and simply forget that you have it installed, but still - would be great to have it somehow implemented in the scripts library from tteck. :)

**Reply by navels** [reply]
totally agree
